fs/expand_glob.ts

```diff
@@ -532,3 +532,29 @@ export function* expandGlobSync(
   }
   yield* currentMatches;
 }
+
+const globEscapeChar = Deno.build.os === "windows" ? "`" : `\\`;
+const globMetachars = "*?{}[]";
```

Comment:
shouldn't we also include `()|+@!`? (Our glob matcher supports extended glob like `@(foo|bar).txt`

Reply:
Good catch. I updated it. I wonder if we should specifically test this unescapeGlobSegment function.